docs/application/native/guides/connectivity/bluetooth.md

```diff
@@ -73,6 +73,8 @@ The main features of the Bluetooth API include:
 
   You can use LE operations to [manage scans](#le_scan), [set scanning filters](#le_scan_filter), [add advertising data](#add_adv_data), [set the advertising connectable mode](#set_adv_conn), [set the advertising mode](#set_adv_mode), and [start and stop advertising](#start_adv).
 
+  You can use extended advertising functionalities such like setting the primary phy, secondary phy and related scan functionalities since tizen 8.0.
```

Comment:
```suggestion
  > [!NOTE]
  > You can use extended advertising functionalities such as setting the primary phy, secondary phy, and related scan functionalities since Tizen 8.0.
```